src/XPurchase/VirtoCommerce.XPurchase/Schemas/WishlistResultType.cs

```diff
@@ -0,0 +1,14 @@
+﻿using GraphQL.Types;
+using VirtoCommerce.ExperienceApiModule.Core.Schemas;
+
+namespace VirtoCommerce.XPurchase.Schemas
+{
+    public class WishlistResultType : ExtendableGraphType<BulkCartResult>
+    {
+        public WishlistResultType()
+        {
+            ExtendableField<CartType>("list", "Wishlst", resolve: context => context.Source.Cart);
+            Field<ListGraphType<ValidationErrorType>>("errors", "Product errors", resolve: context => context.Source.Errors);
```

Comment:
`NonNullGraphType<ListGraphType<NonNullGraphType<ValidationErrorType>>>` if it always return an empty array at least as in other places.